### ISSUE TITLE
Drag Floating Controller down when contain scroll-view cause weird effect

### ISSUE THREAD

**Issue by lucio1392** [SCENEE/FloatingPanel#293]
### Short description

First of all. Thank for very useful and awesome library. 🥰

I have facing with this very weird effect when Floating Controller contain `scroll-view` like `TextView` - `TableView`, etc...

After scroll down for example `text-view`. After that when I try to drag down the Floating Controller, it's very hard to drag down and it's cause some damping effect. It's make a the contain view move up then down.

But when scroll to top of scroll-view then everything become normal.

Please take a look at the video below

https://imgur.com/2NOYwhI

**Code example that reproduces the issue**

Actually I using Map example from FloatingPanel's source. Just create many datasource for tableview inside FloatingController

### Environment

Simulator from iOS 11 -> iOS 13

Real device iPhone XSMax iOS 13.x

**Library version**
1.7.0

I also try with 1.6.6.

**Installation method**

- [x] Direct from source code
- [x] CocoaPods
- [ ] Carthage
- [ ] Git submodules

**iOS version(s)**
from iOS 11 -> iOS 13
**Xcode version**
11.0

**Comment by lucio1392** [comment]
@SCENEE Can you help, please ?

**Comment by SCENEE** [comment]
It seems to be caused by [`grabberAreaFrame ` ](https://github.com/SCENEE/FloatingPanel/blob/master/Framework/Sources/FloatingPanelCore.swift#L264). Could you please reproduce this issue after the `grabberAreaFrame` is set to `.zero`?

**Comment by lucio1392** [comment]
> It seems to be caused by [`grabberAreaFrame ` ](https://github.com/SCENEE/FloatingPanel/blob/master/Framework/Sources/FloatingPanelCore.swift#L264). Could you please reproduce this issue when the `grabberAreaFrame` is set to `.zero`?

 ```
var grabberAreaFrame: CGRect {
        let grabberAreaFrame = CGRect(x: surfaceView.bounds.origin.x,
                                     y: surfaceView.bounds.origin.y,
                                     width: surfaceView.bounds.width,
                                     height: surfaceView.topGrabberBarHeight * 2)
        return .zero
    }
```

I have to set `grabberAreaFrame` to .zero but this problem still occur.

**Comment by SCENEE** [comment]
Thanks for your report. I'm wondering v1.7.1 might be able to fix this issue, but not sure.
I'm sorry I'm not able to figure out gestures to reproduce this issue from your video.

**Comment by SCENEE** [comment]
@lucio1392 Could you make a video also capturing fingers? It's no problem to send it to me by my email.

**Comment by lucio1392** [comment]
Sorry I was to busy with my project. I will try out v1.7.1 first then if the problems still occur I will record the video and sent to you. Thanks.

**Comment by lucio1392** [comment]
Sorry, the problem still happen on newest 1.7.1 version. 

I have sent you an email to contact@scenee.com with video attached. Please help to check. Thanks.

**Comment by SCENEE** [comment]
Hi, I've fixed this issue on #307 😄

**Comment by SCENEE** [comment]
@lucio1392 If you see the reproduction, please feel free to reopen this.

**Comment by lucio1392** [comment]
hi @SCENEE,

Thank for your hard-work. I confirmed that this problem was fixed but look like it's cause another side-effect. 

Sometimes when I try to pull-down during tableview scroll-down ( Maps example ). It's cause the tableview content-offset go out screen's bounds. Please check the image below. Should I open another issue for this problem ?

<img width="1680" alt="Screen Shot 2020-02-06 at 11 18 36" src="https://user-images.githubusercontent.com/13637757/73905692-d1e50280-48d2-11ea-8a01-8214108d6f86.png">

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-06 at 11 17 31](https://user-images.githubusercontent.com/13637757/73905754-05279180-48d3-11ea-87db-e9e4fe373e7a.png)

**Comment by SCENEE** [comment]
Hi @lucio1392,

It looks a side effect by this addition, https://github.com/SCENEE/FloatingPanel/pull/307/commits/14011a5bc28129e4371de802cc1558f0257ba097#diff-e0283e1d782eb6e0ab376186bb1317dbR683.

> Should I open another issue for this problem ?
Yes, please. I would like to take care of this in a new ticket 😄

**Comment by lucio1392** [comment]
> Hi @lucio1392,
> 
> Thank you for your check! It looks a side effect by this addition, [14011a5#diff-e0283e1d782eb6e0ab376186bb1317dbR683](https://github.com/SCENEE/FloatingPanel/commit/14011a5bc28129e4371de802cc1558f0257ba097#diff-e0283e1d782eb6e0ab376186bb1317dbR683).
> 
> > Should I open another issue for this problem ?
> 
> Yes, please. I would like to take care of this in a new ticket 😄
> 
> --
> [Edit]
> 
> I've confirmed the above issue. In fact, the above change I mentioned at link is probably the root cause. So you don't need to create a new issue and we can track it in PR #315.

Thanks!!!

**Comment by SCENEE** [comment]
Hi, @lucio1392. I'm sorry for my late. I fixed the above regression in #315 and merge it to master.

**Comment by lucio1392** [comment]
> Hi, @lucio1392. I'm sorry for my late. I fixed the above regression in #315 and merge it to master.

Really appreciated for you support sir.
